.editorconfig

```diff
@@ -0,0 +1,17 @@
+# EditorConfig is awesome: http://EditorConfig.org
+
+root = true
+
+[*]
+end_of_line = lf
+insert_final_newline = true
+charset = utf-8
+trim_trailing_whitespace = true
+indent_style = space
+indent_size = 2
```

Comment:
We for sure want intend size of 4 for Rust files.

Isn't that the case for most files? I mostly see 2 for xml

Reply:
I assume because noone tried it with Rust yet :)

Rustfmt enforces 4 spaces

Reply:
Honestly, I have no stakes in this game. But as far as can see, the tab vs spaces dispute is over, and spaces clearly won. At least for Rust, using anything else than rustfmt defaults would just needlessly confuse people. To a slightly lesser degree, the same is true for Python and black. In both cases the answer is indentation of 4 spaces.

Reply:
Thanks